dev/make-distribution.sh

```diff
@@ -233,7 +233,7 @@ if [ "$MAKE_PIP" == "true" ]; then
   pushd "$SPARK_HOME/python" > /dev/null
   # Delete the egg info file if it exists, this can cache older setup files.
   rm -rf pyspark.egg-info || echo "No existing egg info file, skipping deletion"
-  python setup.py sdist
+  python3 setup.py sdist
```

Comment:
This looks good itself.

Reply:
Does this mean that `preview2` is the first PySpark distribution packaged by Python3?